rest/rest-sqs-testing-amazon-java-sdk/src/test/scala/org/elasticmq/rest/sqs/AmazonJavaSdkTestSuite.scala

```diff
@@ -894,6 +895,26 @@ class AmazonJavaSdkTestSuite extends FunSuite with Matchers with BeforeAndAfter
     msgIdSent should be(msgIdReceived)
   }
 
+  test("should create queue with redrive policy.") {
+    // When
+    val queueUrl = client.createQueue(new CreateQueueRequest("testQueue1")
+      .withAttributes(Map(
+        defaultVisibilityTimeoutAttribute -> "1",
+        redrivePolicyAttribute ->
+          """
+            |{
+            |  "queueName":"dlq1",
```

Comment:
they do? which ones? I think they should be the same ... if the tested code is supposed to work the same with real and mock SQS :)

can you set the `defaultVisibilityTimeout` when defining a dead letter queue for an existing queue? Or only when creating a queue?